GoogleSignIn/Sources/Public/GoogleSignIn/GIDVerifyAccountDetail.h

```diff
@@ -43,6 +43,21 @@ typedef void (^GIDVerifyCompletion)(GIDVerifiedAccountDetailResult *_Nullable ve
 /// The active configuration for this instance of `GIDVerifyAccountDetail`.
 @property(nonatomic, nullable) GIDConfiguration *configuration;
 
+/// Initialize a `GIDVerifyAccountDetail` object by specifying all available properties.
+///
+/// @param config The configuration to be used.
+/// @return An initialized `GIDVerifyAccountDetail` instance.
+- (instancetype)initWithConfig:(GIDConfiguration *)config
+    NS_DESIGNATED_INITIALIZER;
+
+
+/// Initialize a `GIDVerifyAccountDetail` object by calling the designated initializer 
+/// with the default configuration from the bundle's Info.plist.
+///
+/// @return An initialized `GIDVerifyAccountDetail` instance.
+///     Otherwise, `nil` if the configuration is invalid .
```

Comment:
nit: remove space before period.

Also suggest replacing "Otherwise, `nil` if the configuration is invalid ." with "Otherwise, `nil` if the configuration cannot be automatically generated from your app's Info.plist."

Reply:
Updated!